static/js/stream_edit.js

```diff
@@ -249,9 +249,11 @@ exports.stream_settings = function (sub) {
         const ret = {
             name: setting,
             label: settings_labels[setting],
+            is_disabled: check_realm_setting[setting],
```

Comment:
Can you explain, maybe in the commit message, how `check_realm_setting` corresponds to the previous `realm_setting_disabled` value?  It's not obvious to me without checking out the code.

Reply:
I guess it's clear from the next commit, but I think it's still useful to explain that.